config/routes.rb

```diff
@@ -3,4 +3,32 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'application#welcome'
+
+  # Hotel
+  get '/hotels', to: 'hotels#index', as: 'hotels'
+  get '/hotels/:id', to: 'hotels#show', as: 'hotel'
+  get '/hotels/:id/edit', to: 'hotels#edit', as: 'edit_hotel'
+  patch '/hotels/update', to: 'hotels#update', as: 'update_hotel'
+  delete '/hotels/delete', to: 'hotels#destroy', as: 'delete_hotel'
+
+  # Room
+  get '/rooms', to: 'rooms#index', as: 'rooms'
+  get '/rooms/:id', to: 'rooms#show', as: 'room'
+  get '/rooms/:id/edit', to: 'rooms#edit', as: 'edit_room'
+  patch '/rooms/update', to: 'rooms#update', as: 'update_room'
+  delete '/rooms/delete', to: 'rooms#destroy', as: 'delete_room'
+
+  # Guest
+  get '/guests', to: 'guests#index', as: 'guests'
+  get '/guests/:id', to: 'guests#show', as: 'guest'
+  get '/guests/:id/edit', to: 'guests#edit', as: 'edit_guest'
+  patch '/guests/:id', to: 'guests#update', as: 'update_guest'
```

Comment:
I think I left a comment similar to this before - it is _**not**_ good practice to fill out the routes file with every single  possible route; it leaves your application open to bad actors. From now on, please only fill out this file with the routes that are necessary to fulfill the needs of the user stories. 
This morning you'll learn more about this, and if you want to learn more about it, please ask during that class, and read more in [this article](https://guides.rubyonrails.org/routing.html#restricting-the-routes-created).